FlatRateTaxCalculator.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Price_Calculator_kata
+{
+    public class FlatRateTaxCalculator : ITaxCalculator
+    {
+        private static double _TaxPercentageDefault = 20.0;
```

Comment:
I suggest to move this to a separate config file where all constants/configs can be changed in one place

Reply:
do you mean a  simple text file or a new class for all configurations?